src/com/nisovin/magicspells/CastListener.java

```diff
@@ -61,12 +62,12 @@ public void onPlayerInteract(PlayerInteractEvent event) {
 		if (noInteract) {
 			// special block -- don't do normal interactions
 			noCastUntil.put(event.getPlayer().getName(), System.currentTimeMillis() + 150);
-		} else if (event.getAction() == Action.LEFT_CLICK_AIR || event.getAction() == Action.LEFT_CLICK_BLOCK) {
+		} else if (event.getHand() == EquipmentSlot.HAND && (event.getAction() == Action.LEFT_CLICK_AIR || event.getAction() == Action.LEFT_CLICK_BLOCK)) {
```

Comment:
We can't actually call this directly as not all supported versions have the getHand method in this event.